### ISSUE TITLE
XMLHttpRequest cannot load 'picUrlIsrael' see console.log

### ISSUE THREAD

**Issue by MohamedOmarii** [facn2/knafeh#54]
No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'null' is therefore not allowed access.

**Comment by m4v15** [comment]
Yeah you should have in your readme that this won't work locally without the chrome extension
